lang/cs/.nuget/NuGet.Config

```diff
@@ -21,11 +21,7 @@ under the License.
   <solution>
     <add key="disableSourceControlIntegration" value="true" />
   </solution>
-  <packageSources>
```

Comment:
When I installed xUnit, it asked me to update to later nuget version. Now that I've updated it, nuget does this change automatically at every build. I don't think it makes a lot of sense to keep this file as it was.